apps/chronicles/screens/components/AutocompleteScreen.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from "react";
+import { ScrollView, View } from "react-native";
+import {
+    EDSStyleSheet,
+    useStyles,
+    Typography,
+    Spacer,
+    Autocomplete,
+} from "@equinor/mad-components";
+
+const awesomeAnimals = [
+    "Snow Leopard",
+    "Red Panda",
+    "Platypus",
+    "Narwhal",
+    "Axolotl",
+    "Mantis Shrimp",
+    "Tarsier",
+    "Fennec Fox",
+    "Komodo Dragon",
+    "Okapi",
+    "Aardvark",
+    "Quokka",
+    "Kakapo",
+    "Dik-dik",
+    "Thorny Devil",
+    "Sun Bear",
+    "Gharial",
+    "Toucan",
+    "Mandarinfish",
+    "Proboscis Monkey",
+    "Giant Pangolin",
+    "Flying Fox",
+    "Sloth Bear",
+    "Nudibranch",
+    "Ocelot",
+    "Sugar Glider",
+    "Cassowary",
+    "Lemur",
+    "Peacock Mantis Shrimp",
+    "Tamarin",
+];
+
+export const AutocompleteScreen = () => {
+    const styles = useStyles(themedStyles);
+
+    return (
+        <ScrollView contentInsetAdjustmentBehavior="automatic" contentContainerStyle={{ flex: 1 }}>
+            <View style={styles.container}>
+                <Typography variant="h3">Welcome to the Jungle!</Typography>
+                <Spacer />
+                <Typography>Discover the hidden creatures of the Autocomplete.</Typography>
+                <Spacer />
+                <Autocomplete
+                    options={awesomeAnimals}
+                    label="Click me and spot a Jungle Creature"
+                    placeholder="Type a name and see who's lurking..."
+                    onSelect={selectedOption => console.log(`Spotted: ${selectedOption}`)}
+                />
+            </View>
+            <View style={styles.container}>
+                <Typography>Explore the Jungle Canopy</Typography>
+                <Spacer />
+                <Autocomplete
+                    options={awesomeAnimals}
+                    label="Find a Tree-dweller"
+                    placeholder="Who's swinging in the trees?"
+                    onSelect={selectedOption => console.log(`Found: ${selectedOption}`)}
```

Comment:
We should perhaps remove these console logs. When using the app, you don't see them. If we want to show the user the selections, consider showing it in the app instead. I'm ok with not showing anything as well though. Whatever you want